README.md

```diff
@@ -134,7 +134,6 @@ for transferring data with URLs.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/draios/sysdig"><b>sysdig</b></a> - system exploration and troubleshooting tool with first class support for containers.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://nicolargo.github.io/glances/"><b>glances</b></a> - cross-platform system monitoring tool written in Python.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/hishamhm/htop"><b>htop</b></a> - interactive text-mode process viewer for Unix systems. It aims to be a better 'top'.<br>
-&nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/aksakalli/gtop"><b>gtop</b></a> - system monitoring dashboard for terminal written in JavaScript.<br>
```

Comment:
I think gtop is good but java script? It's lightweight? In addition, it does not display too much information.

Reply:
I am currently surprised at what JavaScript is achieving. It has tools for low-level jobs, such as: analyzing TCP packets.

Reply:
some examples:

- [reverse-shell](https://github.com/lukechilds/reverse-shell)
- [node-aircrack-ng](https://github.com/alnn/node-aircrack-ng#readme)
- [evilredis](https://github.com/matiasinsaurralde/evilredis)